module/product/src/Application/Form/Product/BatchAction/ProductEditForm.php

```diff
@@ -6,44 +6,61 @@
 
 declare(strict_types=1);
 
-namespace Ergonode\BatchAction\Application\Form\Type;
+namespace Ergonode\Product\Application\Form\Product\BatchAction;
 
-use Ergonode\BatchAction\Application\Form\Model\BatchActionPayloadModel;
+use Ergonode\BatchAction\Application\Form\BatchActionFormInterface;
+use Ergonode\BatchAction\Application\Form\Model\BatchActionFormModel;
+use Ergonode\BatchAction\Application\Form\Type\BatchActionFilterType;
+use Ergonode\Product\Application\Form\Product\Attribute\Update\UpdateAttributeValueForm;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\Extension\Core\Type\CollectionType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class BatchActionPayloadType extends AbstractType
+class ProductEditForm extends AbstractType implements BatchActionFormInterface
```

Comment:
Type 'product_edit'
Used
`ProductChangeAttributeBatchActionProcessor`
`ChangeProductAttributesCommand`
I thing is beter type name:
`product_attribute_change `